setup.sh

```diff
@@ -457,11 +488,113 @@ startKuzzle() {
   done
 }
 
-isKuzzleRunning() {
-  write "[ℹ] Checking that everything is running"
+installSystemService() {
+  KUZZLE_ETC_PATH="/etc/kuzzle"
+  SYSTEMD_FILE_PATH="/lib/systemd/system/kuzzle.service"
+  if [[ `systemctl` =~ -\.mount ]]; then
+    echo
+    writeBold "[❓] Would you like to install Kuzzle as a system service? (y/N)"
+    write     "    This will enable you to manage Kuzzle with commands like"
+    write     "    service kuzzle start"
+    write     "    service kuzzle stop"
+    write     "    service kuzzle status"
+    if [ $EUID != 0 ]; then
+      write "[ℹ] You will be probably prompted for your password."
+      SUDO_PREFIX="sudo "
+    fi
+    echo -n "> "
+    read systemdService trash
+      case "$systemdService" in
+        [yY])
+          if [ ! -d $KUZZLE_ETC_PATH ]; then
+            $SUDO_PREFIX mkdir $KUZZLE_ETC_PATH
+          fi
+          $SUDO_PREFIX cp $COMPOSER_YML_PATH $KUZZLE_ETC_PATH/docker-compose.yml
+          SERVICE_FILE_CONTENT=$(cat << EndOfMessage
+[Unit]
+Description=Kuzzle
+After=docker.service
+Requires=docker.service
+
+[Service]
+WorkingDirectory=/etc/kuzzle
+ExecStartPre=-/usr/local/bin/docker-compose kill
+ExecStartPre=-/usr/local/bin/docker-compose rm -f
+ExecStartPre=-/usr/local/bin/docker-compose pull
+ExecStart=/usr/local/bin/docker-compose up
+ExecStop=/usr/local/bin/docker-compose kill
+
+[Install]
+WantedBy=multi-user.target
+EndOfMessage
+)
+          $SUDO_PREFIX tee "$SYSTEMD_FILE_PATH" <<< "$SERVICE_FILE_CONTENT" &> /dev/null
+          $SUDO_PREFIX systemctl enable /lib/systemd/system/kuzzle.service
+          if [ $? -eq 0 ]; then
+            echo
+            writeBold "$GREEN" "[✔] Kuzzle service successfully installed."
+            write     "    Use the following commands to manage Kuzzle"
+            write     "    service kuzzle start"
```

Comment:
`service` is a system V util. It is wrapped to systemctl on debian-like, so it works but is not standard.

`systemctl start|stop|status kuzzle`